src/firefly/java/edu/caltech/ipac/firefly/visualize/WebPlotRequest.java

```diff
@@ -377,6 +378,29 @@ public static WebPlotRequest makeWiseRequest(WorldPt wp,
         return req;
     }
 
+    /**
+     * @param wp
+     * @param survey for atlas, survey is in form of 'schema.table'
+     * @param band SEIP exmaple 'irac1'
+     * @param filter for SEIP, it should loo like type=science and fname like %.mosaic.fits
```

Comment:
I can't find where "type=science and.." is specified.  Is this something that's given via meta, or it's something we need to send along as part of IBE's api?  My concern is where this logic is kept.

Reply:
yes, it will be given from the master table but for FC3 we need to set it in the code. And yes, IBE needs it as part of the query (where clause), different datasets have different filter.

Once we have the framework in place for IV new image search working, and decide to bring the same component to FC, we will need to refactor the FC server code so it can handle generic web plot request and download file from reading the same master table.